docs/userguide/dockervolumes.md

```diff
@@ -216,7 +216,8 @@ then mounting the volumes from the `dbstore` container hides the
 from the `dbstore` container are visible.
 
 You can use multiple `--volumes-from` parameters to bring together multiple data
```

Comment:
@HackToday  A path can be a path to  a file or a path to directory (folder).  Paths, whether they go to files or directories, can be absolute or relative.  Here is a good explanation: http://www.linuxnix.com/abslute-path-vs-relative-path-in-linuxunix/